kore/test/Test/Kore/Attribute/Priority.hs

```diff
@@ -0,0 +1,72 @@
+module Test.Kore.Attribute.Priority where
```

Comment:
I would add a few more test cases:
- negative number as the argument, e.g. "-123"
- string as an argument
- space as an argument